blog/docs/articles/singel-node-kafka-development.md

```diff
@@ -0,0 +1,211 @@
+# Using Apache Kafka for Local Knative Development
+
+**Author: Matthias Weßendorf, Principal Software Engineer @ Red Hat**
+
+**Date: 2023-01-11**
+
+_In this blog post you will learn howto use a production-like environment for your local development with Knative Broker and Apache Kafka._
+
+The [Knative Broker implementation for Apache Kafka](https://knative.dev/docs/eventing/brokers/broker-types/kafka-broker/) is a Kafka-native implementation of the [Knative Broker APIs](https://knative.dev/docs/eventing/brokers/), offering improvements over the usage of the Channel-based Knative Broker implementation, such as reduced network hops, support of any Kafka version and a better integration with Apache Kafka for the Broker and Trigger model.
+
+### Picking the local development enviornment
+
+At the beginning of every project one important aspect is always to keep the development process as simple and also realistic as possible.
+
+When going with the _Knative Broker APIs_ one first thought for the development enviornment could be the usage of the `MTChannelBasedBroker` reference broker (backed by the `InMemoryChannel`) as the main tools for development, because there are no extra systems needed for it to function. 
+
+However, while the APIs for the Knative Broker are generic and of course the same for any broker implementation, there are still some behavioural differences with the different broker implementations. For sure a system like Apache Kafka is way different compared to an "In Memory" storage. 
+
+
+### From start to end with Apache Kafka
+
+When targeting Apache Kafka you should also use it directly for the development environment, while there is some little overhead with Apache Kafka to operate it, there is good news that with [Strimzi operator](https://strimzi.io/) and Apache Kafka in version 3+ it is very easy to use just a single node Kafka cluster for the development, yet being close to a realistic enviornment.
+
+#### No Apache Zookeeper is needed!
+
+This all is possible due to the `KRaft` feature of Kafka. It has been developed over the past years and was recently announced a production feature[](https://www.infoq.com/news/2022/10/apache-kafka-kraft/) of [Apache Kafka 3.3.1](https://archive.apache.org/dist/kafka/3.3.0/RELEASE_NOTES.html).
+
+The Strimzi operator is offering a [FeatureGate](https://strimzi.io/docs/operators/in-development/configuring.html#ref-operator-use-kraft-feature-gate-str) to make use of it, since its version 0.29. KRaft itself is an implementation of the [Raft Consensus Algorithm](https://raft.github.io/) residing directly in [Apache Kafka](https://kafka.apache.org/documentation/#kraft), removing the need for a 3rd-party tool like Zookeeper.
```

Comment:
```suggestion
The Strimzi operator is offering a [FeatureGate](https://strimzi.io/docs/operators/in-development/configuring.html#ref-operator-use-kraft-feature-gate-str) to make use of it, since its version 0.29. `KRaft` itself is an implementation of the [Raft Consensus Algorithm](https://raft.github.io/) residing directly in [Apache Kafka](https://kafka.apache.org/documentation/#kraft), removing the need for a 3rd-party tool like Zookeeper.
```